src/main/webapp/resources/js/pages/user/components/UserProjectsPage.jsx

```diff
@@ -1,10 +1,81 @@
 import React from "react";
+import { useParams } from "react-router-dom";
+import { notification, Space, Switch, Table, Typography } from "antd";
```

Comment:
`Table` not being used

Reply:
Please see 41bc802d312d8a4ecdf45ce51a28db8d2b73bfbf.